Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM starefossen/github-pages:137
 
-COPY . /usr/src/app/
+COPY . /usr/src/app
```

Comment:
It is related. One is what @johndmulhausen told me to use and the other one (without the `/`) is what is in the `onbuild` version of the `github-pages` plugin. Probably it doesn't matter but I thought maybe it was best to stick to how they had it.